src/Core/Traits/HasMeta.php

```diff
@@ -189,7 +189,7 @@ public function syncMeta(array $post)
         $updates = [];
         $deletes = [];
 
-        foreach (setting('Users.metaFields') as $group => $fields) {
+        foreach (setting("{$this->configClass}.metaFields") as $group => $fields) {
```

Comment:
it's possible this will come back as a null value, I believe. So we need some error checking around this it looks like.